inc/class-plugin.php

```diff
@@ -551,7 +551,7 @@ public static function get_attachment_files( int $attachment_id ) : array {
 		$meta = wp_get_attachment_metadata( $attachment_id );
 		if ( isset( $meta['sizes'] ) ) {
 			foreach ( $meta['sizes'] as $size => $sizeinfo ) {
-				$files[] = $uploadpath['basedir'] . $sizeinfo['file'];
+				$files[] = $uploadpath['path'] . '/' . $sizeinfo['file'];
```

Comment:
Hmm `path` will return the current month directory. I guess I had thought that `$sizeinfo['file']` was a relative path to the basedir, but that doesn't seem to be the case. I think it might need to be something like `basedir( $main_file ) . '/' . $sizeinfo['file']`?

Reply:
You were right, I definitely noticed the dates not matching after a while. I had to change the code a bit, but haven't had a chance revisit this ticket yet.